apps/artemis-web/pages/admin/system.tsx

```diff
@@ -63,34 +64,38 @@ const SystemPage = (props) => {
     exabgptap: 'ExaBGP Monitor',
     detection: 'Detection',
     mitigation: 'Mitigation',
+    database: 'Database',
```

Comment:
Why is the database here? It is not a controllable service, it starts by default.

Reply:
Oh no, I was testing something, and forgot to remove this!